slash/css/src/Accordion/Accordion.css

```
@@ -0,0 +1,105 @@
+.af-accordion {
+  font-family: var(
+    --font-family-sans-serif,
+    "Source Sans Pro",
+    arial,
+    sans-serif
+  );
+}
+
+.af-accordion__item-header {
+  display: flex;
+  min-height: 3rem;
+  padding: 0 1.25rem;
+  align-items: center;
+  justify-content: space-between;
+  color: var(--white);
+  background-color: var(--axablue80);
+  cursor: pointer;
+  list-style-type: none;
+}
+
+.af-accordion__item-header-icon {
+  width: 2rem;
+  height: 2rem;
+  order: 2;
+  transition: transform 0.5s;
+  fill: currentcolor;
+}
+
+.af-accordion__details {
+  margin-bottom: 1px;
+}
+
+.af-accordion__item-title {
+  margin: 0;
+  order: 1;
+  font-size: calc(20rem / 16);
+}
+
+.af-accordion__content {
+  padding: 1rem;
+  box-shadow: 0 0 9px 0 rgba(0, 0, 0, 18%);
+}
+
+.af-accordion__details[open] .af-accordion__item-header-icon {
+  transform: rotate(180deg);
+}
+
+.af-accordion__header-right {
+  display: flex;
+  align-items: center;
+  gap: 1rem;
+}
+
+.af-accordion--light .af-accordion__item-header {
+  display: block;
+  padding: 0;
+  background: transparent;
+}
+
+.af-accordion--light .af-accordion__item-header svg {
```

tu pouvais quand même faire du nesting css
```suggestion
.af-accordion--light  {
   .af-accordion__item-header {
        svg {
        
        }
        
        button {
           &:first-of-type {
           
           }
        }
        
```

---

je l'ai fait sur accordéon, mais je ne vais pas repasser sur toute la codebase finalement. Je peux logguer une issue pour le faire par contre